src/category_theory/connected.lean

```diff
@@ -221,4 +221,42 @@ lemma nat_trans_from_connected [conn : connected J] {X Y : C}
   (λ j, α.app j)
   (λ _ _ f, (by { have := α.naturality f, erw [id_comp, comp_id] at this, exact this.symm }))
 
+omit 𝒞 𝒥
+section examples
```

Comment:
What's the rationale for moving these examples back here? If we could have `category_theory.connected` not have to import anything from the `category_theory.limits` directory, that would be great. (For the long term, I think carefully minimising imports will pay off.)

Reply:
It doesn't import from category_theory.limits :)
It's also because connected categories aren't strictly related to limits, so having examples in that folder is a little odd.

Reply:
Also, the example instances I give need to use that the category is connected, so we'd need to import from category_theory.limits.connected at the top of limits.over. 

That said, if you prefer I move this new proof into limits.connected, I'd be okay with that.

Reply:
I don't understand: at the top of this file is
```lean
import category_theory.limits.shapes.pullbacks
import category_theory.limits.shapes.binary_products
import category_theory.limits.shapes.equalizers
import category_theory.limits.preserves
```

Reply:
Maybe you're looking at a different file?

Reply:
Entirely my mistake, I'll put them back in `limits.connected`.